resolwe/test/testcases/process.py

```diff
@@ -253,7 +253,10 @@ def setUp(self):
         """Initialize test data."""
         super().setUp()
 
-        self._register_schemas()
+        _, stderr = self._register_schemas()
+        stderr = stderr.getvalue()
+        if stderr:
+            raise ValueError(stderr)
```

Comment:
This is not `ValueError`, more `RuntimeError`. Otherwise, it looks good.

Reply:
Fixed.